client/reader/stream/index.jsx

```diff
@@ -438,6 +503,22 @@ class ReaderStream extends React.Component {
 				/>;
 		}
 
+		if ( postKey.isCombination ) {
+			const handleClick = post => showSelectedPost( {
```

Comment:
this has the same problem as before :D you're making a new function for every combined card every time render runs. That means the combined card is getting different props and cannot take advantage of `shouldComponentUpdate` to avoid re-rendering.